hera_mc/autocorrelations.py

```diff
@@ -75,7 +75,8 @@ def __repr__(self):
 
 
 def plot_HERA_autocorrelations_for_plotly(session):
-    from plotly import graph_objs as go, plotly as py
+    from plotly import graph_objects as go
+    from chart_studio import plotly as py
 
     hera_ants = [9, 10, 20, 22, 31, 43, 53, 64, 65, 72, 80, 81, 88, 89, 96, 97, 104, 105, 112]
```

Comment:
Is there a way we can get this out of config management rather than hardcoding it?

Reply:
oh probably. I didn't even look at that line honestly just went on replacing the plotly import. I'l see what I can do with these hard-coded antennas.